Documentation/community/roadmap.rst

```diff
@@ -14,35 +14,60 @@ broader community. You'll also find here some pointers on how you can
-|| Kube-proxy Replacement     | Stable                                                     |
-++----------------------------+------------------------------------------------------------+
-|| Egress Gateway             | Stable                                                     |
-++----------------------------+------------------------------------------------------------+
-| Multi-Cluster (ClusterMesh) | Stable (:ref:`Roadmap Details<rm-clustermesh>`)            |
-+-----------------------------+------------------------------------------------------------+
-| Hubble Observability        | Stable (:ref:`Roadmap Details<rm-hubble-observability>`)   |
-+-----------------------------+------------------------------------------------------------+
-| Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
-+-----------------------------+------------------------------------------------------------+
-| Tetragon Security           | Beta                                                       |
-+-----------------------------+------------------------------------------------------------+
+
++--------------------------------------------------+----------------------------------------------------------+
+| eBPF Networking                                  | Stable (:ref:`Roadmap Details<rm-advanced-networking>`)  |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Kubernetes CNI<k8s_network_root>`         | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| `Load Balancing`_                               | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`k8s_policy`                               | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Kube-proxy Replacement<kubeproxy-free>`   | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`egress-gateway`                           | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gsg_encryption` (IPSec and WireGuard)     | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`bandwidth-manager`                        | Stable                                                   |
```

Comment:
Do we have input from users on stability of bandwidth manager? Are users enabling it? cc @cilium/sig-datapath

Reply:
FWIW it's not currently marked as beta in the [docs](https://docs.cilium.io/en/v1.14/network/kubernetes/bandwidth-manager/) (but that could be an oversight?)

Reply:
@borkmann what's your view, should this be marked as stable or beta?

Reply:
We have 2 enterprise users that use the feature according to internal data, including IKEA (openly states this on the USERS.md)

Reply:
Note: the Bandwidth Manager was publicly promoted to “stable” [in Cilium 1.12's announcement blog](https://isovalent.com/blog/post/cilium-release-112/#bm-stable).

Reply:
Sounds clear that we should be treating it as stable